app/assets/javascripts/services/settingsSvc.js

```diff
@@ -83,6 +81,7 @@ angular.module('QuepidApp')
 
           escapeQuery:       true,
           apiMethod:         'POST',
+          customHeaders:     '',
```

Comment:
why add it here if we remove it above?  In some places we do a check for either for existence of customheaders or an actual check of if it is `''`, are we maybe missing that someplace?